### ISSUE TITLE
option to prevent the aggregation of age groups (in case of sparse data)

### ISSUE THREAD

**Issue by lwillem** [epiforecasts/socialmixr#144]
The current implementation of `contact_matrix()` aggregates the oldest age groups, for example, combining the 60-89 and 90+ age groups into a single "60+" category if there is no data for the 90+ group. More specifically:

------------------
#line 275
adjust age.group.brakes to the lower and upper ages in the survey
  survey$participants[, lower.age.limit := reduce_agegroups(
    get(columns[["participant.age"]]),
    age.limits[age.limits < max.age]
  )]
  part.age.group.breaks <- c(age.limits[age.limits < max.age], max.age)
  part.age.group.present <- age.limits[age.limits < max.age]
------------------

Would it be useful to include a flag to disable this behavior and instead return a matrix with `NA` values if needed, while keeping the age groups fixed? This would be helpful in cases where post-processing assumes a matrix with a fixed number of columns (N), as receiving one with N-1 columns could cause issues.

**Comment by sbfnk** [comment]
Yes, and I think perhaps your suggested behaviour would actually make sense as default.
